app/pages/order-dashboard/controller.js

```diff
@@ -0,0 +1,3 @@
+import { getContext } from './contextCreator';
+
+export const getNewOrderPageContext = () => getContext({ pageName: 'neworder' });
```

Comment:
is the `pageName` here so you can reuse this component?

Reply:
yep. we'll be using the same template for existing order dashboard page